### ISSUE TITLE
Source of BS data?

### ISSUE THREAD

**Issue by munikarmanish** [tnagorra/nepalical#5]
Hi, can you provide the source of the BS data (list of number of days in every BS year/month) ?

**Comment by rhoit** [comment]
https://github.com/tnagorra/nepalical/blob/master/src/nepalical/BsData.cpp#L7-L99

**Comment by munikarmanish** [comment]
I meant the source, not the data. :wink:

**Comment by rhoit** [comment]
data has been floating around for a while, have tried to complie list in https://github.com/foss-np/miti

according to my knowledge data was extracted from the locally available calendar (पात्रो).

**Comment by sarojpoudel33** [comment]
Is this Open Sources. Can we Re-write and make plugin for our application. https://github.com/linuxdeepin/deepin-calendar / https://cr.deepin.io/#/admin/projects/deepin-calendar

**Comment by rhoit** [comment]
yes its possible! why not start with a new project @sarojpoudel33, we have become, quite buzy unable take new projects. 😞

**Comment by sarojpoudel33** [comment]
you can check out those project

**Comment by sarojpoudel33** [comment]
![deepinscreenshot_google-chrome_20181226064002](https://user-images.githubusercontent.com/45546591/50430295-3f122000-08eb-11e9-96a9-10929ca1b35d.png)

**Comment by sarojpoudel33** [comment]
Language Contribute garne manche ne chaiyekko xa.

**Comment by tnagorra** [comment]
@munikarmanish I don't really remember but it was from one of the open source projects

**Comment by sarojpoudel33** [comment]
https://github.com/linuxdeepin/developer-center/issues/790
